src/assets/translations/fr/main.json

```diff
@@ -1554,7 +1565,8 @@
       "agreeItem4": "En activant le Snap ShapeShift Multichain, vous reconnaissez que vous l'utilisez à vos propres risques.",
       "readAndUnderstood": "J'ai lu et compris",
       "seedBackedUp": "J'ai sauvegardé ma phrase de récupération MetaMask",
-      "acceptInstall": "Confirmer & Installer"
+      "acceptInstall": "Confirmer & Installer",
+      "acceptUpdate": "Confirmer et mettre à jour"
```

Comment:
nitpick: inconsistent

Reply:
Thanks for the review, you're right! I've updated few more cases. Only case where I've left an *inconsistent* upper case is for `Feedback et Support", as top level menu it seems better this way.